### PR TITLE
Adressing EDPM issues

### PR DESCRIPTION
**Pull request by fao89**

### CODE REVIEW

File: scripts/gen-edpm-kustomize.sh [hunk 1]
```diff
@@ -76,9 +76,6 @@ patches:
     - op: replace
       path: /spec/nodeTemplate/nodes/edpm-compute-0/ansible/ansibleVars/ctlplane_ip
       value: ${EDPM_COMPUTE_IP}
-    - op: replace
-      path: /spec/nodeTemplate/nodes/edpm-compute-0/ansibleSSHPrivateKeySecret
```
Comment (fao89) [inline scripts/gen-edpm-kustomize.sh:80]: secret is not set in the nodes individually anymore

**Comment by fao89** [comment]
recheck

**Comment by fao89** [comment]
recheck

**Comment by fao89** [comment]
recheck

**Review by rabi — APPROVED** [review]
/lgtm

**Pull request merged**